assets/sass/modules/_forgot-password.scss

```diff
@@ -35,7 +35,7 @@
     }
   }
 
-  .o-form-input {
+  .o-form-fieldset.enroll-sms-phone {
```

Comment:
what about `enroll-call-phone`? we need to fix that too

Reply:
They use the same element with class `enroll-sms-phone`, is a unique controller, see the screenshots.

Reply:
ok...  And this is under `recovery-challenge` so all good  👍